src/app/shared/components/case-actions/case-actions.component.html

```diff
@@ -4,6 +4,8 @@
             <h1 data-selector="heading" role="heading" aria-level="1" class="hmcts-page-heading__title govuk-heading-xl">{{header}}</h1>
         </div>
         <!-- TODO: Should use HMCTS Menu Component here, instead of this. -->
+        <div [unlessRole]="'caseworker-sscs-panelmember'"  roleList="{{roleList}}" roleSource="inline">
```

Comment:
just curious, do we always need to pass the roleList in? Perhaps there's another way the directive can obtain the roles (just means less maintenance for future code)